cmd/create_test.go

```diff
@@ -0,0 +1,192 @@
+package cmd_test
+
+import (
+	"os"
+	"strings"
+
+	"github.com/nimblehq/gin-templates/tests"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = BeforeEach(func() {
+	tests.ChangeDirectory(TemplateGeneratedPath + "/test-gin-templates")
+})
+
+var _ = Describe("Create template", func() {
+	Context("given the project directory", func() {
+		It("contains project name", func() {
+			dir, err := os.Getwd()
+			if err != nil {
+				Fail("Failed to get current directory: " + err.Error())
+			}
+
+			directoryContinProjectName := strings.Contains(dir, "gin-templates/cmd/test-gin-templates")
```

Comment:
```suggestion
			directoryContainProjectName := strings.Contains(dir, "gin-templates/cmd/test-gin-templates")
```

Reply:
Good catch. Fixed in 8a02884b1579d1ef7d9c70e69a90e5099de6a780